next/components/forms/steps/Summary/SummaryFormControls.tsx

```diff
@@ -44,8 +44,8 @@ const SummaryFormControls = () => {
         </div>
 
         <div className="flex flex-row flex-wrap gap-5">
-          {/* Temporary logic for tax form, will be cleaned up. */}
-          {isTaxForm ? (
+          {/* Temporary logic for signed forms, will be cleaned up. */}
+          {isSigned ? (
```

Comment:
This is what I am already working on at https://github.com/bratislava/private-konto.bratislava.sk/issues/721